pkgs/applications/science/math/weka/default.nix

```diff
@@ -1,29 +1,30 @@
 { stdenv, fetchurl, jre, unzip }:
 
-stdenv.mkDerivation {
-  name = "weka-3.6.12";
-  
+stdenv.mkDerivation rec {
+  name = "weka-${version}";
+  version = "3-8-0";
```

Comment:
Please use `3.8.0`; to reduce duplication, you can do `replaceChars ["."] ["-"]`.